platform_mock/platform_mock.py

```diff
@@ -233,13 +233,39 @@ def data_received(self, chunk):
 
     def get(self):
         """Answer GET request"""
-        self.write(json.dumps({'meta': {'count': 1, 'limit': 10, 'offset': 0},
-                               'links': {'first': '/api/rbac/v1/access/?application=vulnerability&limit=1000&offset=0',
-                                         'next': None, 'previous': None,
-                                         'last': '/api/rbac/v1/access/?application=vulnerability,inventory&limit=1000&offset=0'},
-                               'data': [{'permission': 'vulnerability:*:*', 'resourceDefinitions': []},
-                                        {'permission': 'inventory:hosts:read', 'resourceDefinitions': []},
-                                        {'permission': 'remediations:remediation:read', 'resourceDefinitions': []}]}))
+        self.write(
+            json.dumps(
+                {
+                    "meta": {"count": 1, "limit": 10, "offset": 0},
+                    "links": {
+                        "first": "/api/rbac/v1/access/?application=vulnerability&limit=1000&offset=0",
+                        "next": None,
+                        "previous": None,
+                        "last": "/api/rbac/v1/access/?application=vulnerability,inventory&limit=1000&offset=0",
+                    },
+                    "data": [
+                        {
+                            "permission": "inventory:hosts:read",
+                            "resourceDefinitions": [
+                                {
+                                    "attributeFilter": {
+                                        "key": "group.id",
+                                        "value": [
+                                            "00000000-1111-0000-0000-000000000000",  # Group 1
+                                            "00000000-2222-0000-0000-000000000000",  # Group 2
+                                            "null",  # No group (ungrouped hosts)
```

Comment:
according to doc it is not `"null"` string but `null` value, which should be translated as `None` in Python